tests/semantic/ignore_files.json

```diff
@@ -0,0 +1,542 @@
+    "solidity/test/libsolidity/semanticTests/events/event_indexed_mixed.sol",
+    "solidity/test/libsolidity/semanticTests/events/event_indexed_string.sol",
+    "solidity/test/libsolidity/semanticTests/events/event_lots_of_data.sol",
+    "solidity/test/libsolidity/semanticTests/events/event_lots_of_data.sol",
+    "solidity/test/libsolidity/semanticTests/events/event_no_arguments.sol",
+    "solidity/test/libsolidity/semanticTests/events/event_really_lots_of_data.sol",
+    "solidity/test/libsolidity/semanticTests/events/event_really_lots_of_data_from_storage.sol",
+    "solidity/test/libsolidity/semanticTests/events/event_really_really_lots_of_data_from_storage.sol",
+    "solidity/test/libsolidity/semanticTests/events/event_signature_in_library.sol",
+    "solidity/test/libsolidity/semanticTests/events/event_string.sol",
+    "solidity/test/libsolidity/semanticTests/events/event_struct_memory_v2.sol",
+    "solidity/test/libsolidity/semanticTests/events/event_struct_storage_v2.sol",
+    "solidity/test/libsolidity/semanticTests/events/events_with_same_name.sol",
+    "solidity/test/libsolidity/semanticTests/events/events_with_same_name_inherited_emit.sol",
+    "solidity/test/libsolidity/semanticTests/externalContracts/deposit_contract.sol",
+    "solidity/test/libsolidity/semanticTests/functionCall/external_call_value.sol",
+    "solidity/test/libsolidity/semanticTests/functionCall/value_test.sol",
+    "solidity/test/libsolidity/semanticTests/functionTypes/inline_array_with_value_call_option.sol",
+    "solidity/test/libsolidity/semanticTests/modifiers/function_modifier.sol",
+    "solidity/test/libsolidity/semanticTests/payable/no_nonpayable_circumvention_by_modifier.sol",
+    "solidity/test/libsolidity/semanticTests/smoke/basic.sol",
+    "solidity/test/libsolidity/semanticTests/smoke/fallback.sol",
+
+    "// L1 builtins",
+    "solidity/test/libsolidity/semanticTests/builtinFunctions/blockhash.sol",
+    "solidity/test/libsolidity/semanticTests/builtinFunctions/blockhash_shadow_resolution.sol",
+    "solidity/test/libsolidity/semanticTests/builtinFunctions/msg_sig.sol",
+    "solidity/test/libsolidity/semanticTests/builtinFunctions/msg_sig_after_internal_call_is_same.sol",
+    "solidity/test/libsolidity/semanticTests/builtinFunctions/ripemd160_empty.sol",
+    "solidity/test/libsolidity/semanticTests/inlineAssembly/basefee_berlin_function.sol",
+    "solidity/test/libsolidity/semanticTests/inlineAssembly/chainid.sol",
+    "solidity/test/libsolidity/semanticTests/state/block_basefee.sol",
+    "solidity/test/libsolidity/semanticTests/state/block_chainid.sol",
+    "solidity/test/libsolidity/semanticTests/state/block_coinbase.sol",
+    "solidity/test/libsolidity/semanticTests/state/block_difficulty.sol",
+    "solidity/test/libsolidity/semanticTests/state/block_gaslimit.sol",
+    "solidity/test/libsolidity/semanticTests/state/block_number.sol",
+    "solidity/test/libsolidity/semanticTests/state/block_timestamp.sol",
```

Comment:
Hmm, can we remove these from the ignore list now that Greg has merged the PR that adds them?

Reply:
@JorikSchellekens is working on running all the ignored files and un-ignoring them